lblod/frontend-lokaal-mandatenbeheer
# Ben/lmb 730 bulk besluiten

## Description

Add a page with the possibility to bulk bekrachtig multiple mandatees at once. All mandatees are mentioned here, also multiple states of the same one, because you need to be able to handle them independently. It does not matter which publication status the mandataris had before, you are able to set it to bekrachtigd, the only thing you can't do is make a mandataris that is bekrachtigd effectief.

![Screenshot 2024-10-24 at 17 14 30](https://github.com/user-attachments/assets/bf73e6b1-5aee-47f4-81ef-f3be71c0899d)


## How to test

Go to the page, it is nested under the orgaan page. Play around with it a bit and see if it does what you expect it to do.

## Links to other PR's

- https://github.com/lblod/mandataris-service/pull/57
